runtime/runtime.go

```diff
@@ -110,7 +110,7 @@ func initializeModules() []primitives.Module {
 		timestamp.NewConfig(auraModule, DbWeight, TimestampMinimumPeriod),
 	)
 
-	grandpaModule := grandpa.New(GrandpaIndex)
+	grandpaModule := grandpa.New[primitives.Ed25519Signer](GrandpaIndex)
```

Comment:
You can define `type SignerKeyType = primitives.Ed25519Signer` in one place to make it easily configurable and use it:
`grandpaModule := grandpa.New[SignerKeyType](GrandpaIndex)`